torch/testing/_internal/common_methods_invocations.py

```diff
@@ -587,6 +603,16 @@ def sample_inputs_pinverse(op_info, device, dtype, requires_grad=False):
                        SkipInfo('TestCommon', 'test_variant_consistency_jit',
                                 device_type='cuda', dtypes=[torch.bfloat16]),
                    )),
+    OpInfo('broadcast_to',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           supports_tensor_out=False,
+           test_inplace_grad=False,
+           sample_inputs_func=sample_inputs_broadcast_to,
+           skips=(
+               # RuntimeError: "isfinite" not implemented for 'BFloat16'
```

Comment:
After a rebase this skip should no longer be necessary - would you try removing it?

Reply:
Thanks so much for pointing this out, `skips` has been removed and the Opinfo-based `broacast_to` tests are passed.